controllers/controllervm_controller.go

```diff
@@ -129,7 +129,7 @@ func (r *ControllerVMReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error
 			Namespace:      instance.Namespace,
 			Type:           common.TemplateTypeConfig,
```

Comment:
Should this be `common.TemplateTypeNone`, as per our conversation last week?  I knew you said you were adding a new type, but I know this works anyhow.

Reply:
yes, good catch, thanks!